Ginger/GingerCoreNET/ALMLib/JIRA/Bll/JiraExportManager.cs

```diff
@@ -369,6 +372,22 @@ public bool ExecuteDataToJira(BusinessFlow bizFlow, PublishToALMConfig publishTo
                                 System.IO.File.Delete(zipFileName);
                             }
                         }
+
+
+                        if (publishToALMConfig.ToExportReportLink)
+                        {
+                            if (!string.IsNullOrEmpty(publishToALMConfig.HtmlReportUrl))
+                            {
+                                string reportLink = publishToALMConfig.HtmlReportUrl + "#/BusinessFlow/" + $"{ publishToALMConfig.ExecutionId}/" + bizFlow.InstanceGuid;
```

Comment:
Use the common function to create report URL

Reply:
handled